### DISCUSSION TITLE
Put Winget in Sandbox by Default

### DISCUSSION THREAD

**Discussion by Trenly** [microsoft/winget-pkgs#56530] [Show and tell]
This powershell script modifies the Base Image, or the Virtual Hard Disk, which the Windows Sandbox launches upon startup. It will copy the required files to the sandbox and add a registry key which will install them upon startup. By default the script will install the latest stable release of Winget. You can specify to use the latest pre-release with the -PreRelease switch.

When a new version of Winget is released, run this script again to update the installation in the sandbox to the latest version.

https://gist.github.com/Trenly/3e8ba9a9498c6cc12a9bb25e4179a98c

> *I did test the compatibility of this with the `SandboxTest.ps1` script. The SandboxTest.ps1 script will install Winget before the registry key attempts to install, and so the SandboxTest.ps1 script still functions as expected*

**Comment by JonathanPitre** [comment]
Can I uptove this by a 1000% ? Sandbox is a wonderful tool to validate install packages such as winget, intune, psadt etc.  It should be coded with that mind or at least give us an option in the UI to enable winget support. Please!

**Reply by denelon** [reply]
We've been working with the Sandbox team on some improvements including figuring out a way to have the App Installer (including WinGet) from the host included in the guest. There isn't an ETA, but I wanted to share a quick update.
